### PR TITLE
feat: Add Fuel and EV methods

### PR DESCRIPTION
**Pull request by KarthikBhaskara**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ smartcar.isExpired = function(expiration) {
  * @param {Number} [paging.limit] - number of vehicles to return
  * @param {Number} [paging.offset] - index to start vehicle list
  * @return {Promise.<module:smartcar~VehicleIds>} A promise with the vehicle ids.
+ * @throws {SmartcarError}
```
Comment (morgannewman) [inline index.js]: I think we should be super explicit for all these `@throws` jsdoc comments.
```suggestion
 * @throws {SmartcarError} - an instance of SmartcarError. See the [errors section of the README](https://github.com/smartcar/node-sdk/tree/master/doc#errors) for all possible errors.
```

If we want to be extra spicy, we can even have multiple `@throws` in the jsdoc for methods that call for it (e.g. FuEV endpoints that can throw VehicleNotCapable and SmartcarNotCapable).

```
 * @throws {SmartcarError} - an instance of SmartcarError. See the [errors section of the README](https://github.com/smartcar/node-sdk/tree/master/doc#errors) for all possible errors.
 * @throws {VehicleNotCapableError} - description...
 * @throws {SmartcarNotCapableError} - description...
```

**Comment by KarthikBhaskara** [comment]
> Super super nit comments, sorry. 
> 
> Because you're removing `NotCapableError`, I believe this is a breaking change. Bumping the major version will be kind of annoying since you'd need to ammend a commit due to [how semantic-release works](https://github.com/semantic-release/commit-analyzer#usage), but I do think we should do it.

Will major bump. don't need to amend, i'll just set the appropriate message before squashing and merging.

File: index.js [hunk 2]
```diff
@@ -76,6 +76,9 @@ smartcar.isExpired = function(expiration) {
  * @param {Number} [paging.limit] - number of vehicles to return
  * @param {Number} [paging.offset] - index to start vehicle list
  * @return {Promise.<module:smartcar~VehicleIds>} A promise with the vehicle ids.
+ * @throws {SmartcarError} - an instance of SmartcarError.
```
Comment (morgannewman) [inline index.js]: I notice this isn't showing up correctly on GitHub when you view the rendered version. `Throws` is on the same line as the return statement. Can you investigate?

Comment (KarthikBhaskara) [reply index.js]: mm that's just how our tool renders it. feel free to add another task to investigate. I just ran `npm run docs`. most docs are on the same line unfortunately :(

Comment (morgannewman) [reply index.js]: Roger that

**Pull request merged**